starter-code/index.html

```diff
@@ -9,6 +9,30 @@
   <script type="text/javascript" src="./js/index.js"></script>
   <title>Ironhack cart</title>
 </head>
-<body>
+<body onload="getPriceByProduct()">
+    <div class="wrapper">
+      <h1 class="title">Ironhack Merchandising Shop</h1>
+      <div>
+        <span class="product-name">Ironhack Cup</span>
+      </div>
+      <form id="price">
+        <input value="10 Euro" type="submit">
+      </form>
```

Comment:
Care using an `input` inside a `form` since when you click on it has a default behaviour.